src/components/footer/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, useMemo } from 'react';
+import { Container } from '@chakra-ui/react';
```

Comment:
Does it make senes to extract the `Container` component?

Reply:
See https://github.com/smg-automotive/components-pkg/pull/349#discussion_r1029208481